context.go

```diff
@@ -151,17 +152,20 @@ func (r *ResponseData) Written() bool {
 // encoders. It uses the default service encoder if no match is found.
 func (r *ResponseData) Send(ctx context.Context, code int, body interface{}) error {
 	r.WriteHeader(code)
+	go IncrCounter([]string{"goa", "response", "manual", strconv.Itoa(code)}, 1.0)
```

Comment:
Would it make more sense to have these in `ReponseData.WriteHeader`? This would capture all responses.